### DISCUSSION TITLE
Texture not showing when bounds="legacy" ?

### DISCUSSION THREAD

**Discussion by clochardM33** [google/model-viewer#3568] [Q&A] [answered]
Hi,
Attached is a model of a cube with a plain material and a transparent png.
When bounds="tight" the png is there, when bounds="legacy" the png isn't there. (More simply bounds is not specified)
Couldn't see any obvious discussion of this already.
Is this supposed to be like this?
Thanks.

[Cube_with_texture.glb.txt](https://github.com/google/model-viewer/files/9048074/Cube_with_texture.glb.txt)

**Comment by elalish** [comment, marked as answer]
Really everyone should use `bounds="tight"` as the legacy version has bugs; this is likely one. In v2.0 we're going to remove the legacy option for this reason.

**Reply by clochardM33** [reply]
Ok. Bounds tight it is then. Thanks.
